src/v/cluster/node/local_monitor.h

```diff
@@ -56,6 +59,9 @@ class local_monitor {
     static size_t
     alert_percent_in_bytes(unsigned alert_percent, size_t bytes_available);
 
+    /// Periodically check node status until stopped by abort source
+    ss::future<> _update_loop();
```

Comment:
nit: what's the significance of the leading underscore vs just making this private?

Reply:
It's a poorly-enforced convention, where we use leading underscores for private functions... sometimes.  We should probably formalize it or stop doing it, really.